ui/src/app/shared/conditions-panel.tsx

```diff
@@ -26,6 +22,16 @@ export function hasWarningConditionBadge(conditions: Condition[]): boolean {
     return false;
 }
 
+export function hasArtifactGCError(conditions: Condition[]): boolean {
+    for (const condition of conditions) {
+        if (condition.type === 'ArtifactGCError') {
```

Comment:
Nitpick. If we want to report all errors (not just ArtifactGCError), this function could be
```
function getErrorMessages(conditions: Condition[]): Error[] {
    let errs = Error[]
    for (const condition of conditions) {
        if (ErrorConditions.includes(condition.type) {
            errs.push(ErrorMessages[condition.type]);
        }
    }
    return errs;
}
```